src/DependencyInjection/Configuration.php

```diff
@@ -104,6 +109,104 @@ public function getConfigTreeBuilder(): TreeBuilder
+                                            ->defaultValue([])
+                                            ->scalarPrototype()->end()
+                                    ->end()
+                                    ->enumNode('query_reject_condition')
+                                        ->values([
+                                            QueryRejectCondition::QUERY_REJECT_CONDITION_UNSPECIFIED,
+                                            QueryRejectCondition::QUERY_REJECT_CONDITION_NONE,
+                                            QueryRejectCondition::QUERY_REJECT_CONDITION_NOT_OPEN,
+                                            QueryRejectCondition::QUERY_REJECT_CONDITION_NOT_COMPLETED_CLEANLY,
+                                        ])
+                                        ->validate()
+                                            ->ifNotInArray([
+                                                QueryRejectCondition::QUERY_REJECT_CONDITION_UNSPECIFIED,
+                                                QueryRejectCondition::QUERY_REJECT_CONDITION_NONE,
+                                                QueryRejectCondition::QUERY_REJECT_CONDITION_NOT_OPEN,
+                                                QueryRejectCondition::QUERY_REJECT_CONDITION_NOT_COMPLETED_CLEANLY,
+                                            ])
+                                            ->thenInvalid(\sprintf('"queryRejectionCondition" value is not in the enum: %s', QueryRejectCondition::class))
+                                        ->end()
+                                    ->end()
+                                ->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('workers')
+                            ->defaultValue([
+                                'default' => [
+                                    'queue' => 'default',
+                                    'exception_interceptor' => 'temporal.exception_interceptor',
+                                    'options' => [],
+                                    'default_interceptors' => true,
+                                    'interceptors' => [],
+                                ],
+                            ])
+                            ->useAttributeAsKey('name')
+                            ->arrayPrototype()
+                                ->addDefaultsIfNotSet()
+                                ->children()
+                                    ->scalarNode('queue')->defaultValue('default')->end()
+                                    ->scalarNode('exception_interceptor')->defaultValue('temporal.exception_interceptor')->end()
+                                    ->arrayNode('options')
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->integerNode('max_concurrent_activity_execution_size')->defaultValue(1)->end()
```

Comment:
consider adding validation for integer nodes that are required to be at least 0
```suggestion
                                            ->integerNode('max_concurrent_activity_execution_size')->min(0)->defaultValue(1)->end()
```